### PR TITLE
[REVIEW] Parquet writer int96 [skip ci]

### PR DESCRIPTION
**Pull request by razajafri**
This PR is dependent upon #5096.

Added changes to the following files
ParquetWriterOptions.java
TableJni.cpp
TableTest.java
Table.java

### CODE REVIEW

**Comment by razajafri** [comment]
@revans2 I had to close the older PR as it was showing many more changes than it should. I have addressed your concerns

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by harrism** [comment]
I think it's dependent on #6625 (PR), not #5096 (issue).

**Review by revans2 — APPROVED** [review]

**Comment by revans2** [comment]
Not sure why the changelog check failed, it is in there.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## New Features
 
 - PR #6116 Add `filters` parameter to Python `read_orc` function or filtering
+- PR #6679 Added Java bindings for writing parquet files with INT96 timestamps
```
Comment (jlowe) [inline CHANGELOG.md:6]: changelog number is wrong

Comment (razajafri) [reply CHANGELOG.md]: :bow:

**Pull request merged**